hyperwallet/tests/test_api.py

```diff
@@ -1324,6 +1324,35 @@ def test_create_paypal_account_success(self, mock_post):
 
         self.assertTrue(response.email, paypal_account_data.get('token'))
 
+    def test_update_paypal_account_fail_need_user_token(self):
```

Comment:
do we have something like parametrized test to avoid duplication? or can you think of any other way?

Reply:
For now in test_api.py for each api endpoint we have a block of tests that covers specified endpoint. So I think if we'll rewrite some tests with parametrized approach, this structure will be destroyed, and the tests for one endpoint will be located in different places, and it would be harder to track all test cases for given endpoint. So I think its better to keep endpoint tests in one place and keep the current tests structure